### ISSUE TITLE
SSC/Vashj - Mind Control

### ISSUE THREAD

**Issue by smkrus** [Looking4Group/L4G_Core#2833]
https://www.youtube.com/watch?v=Udlfkvaq1F8 (I think this is sped up to 1.5x)
Suggests 2x players not CCable
On Hellfire it is every 120 seconds 5x players CCable.
On top of being able to CC. It appears casters all use rank 1 spells.

**Comment by AnonXS** [comment]
From https://youtu.be/SYypbzs9d44?t=8m54s it is possible to chain cc 2 players (which should not be ccable).

9:11 first MC after ~ 15sec , 10:55 MC ends 11:01 New MC

11:44 end 11:48 new Question is Videospeed = x2?

I suggest to put http://db.looking4group.eu/?spell=38511

Initial timer to 15sec from 30sec

https://github.com/Looking4Group/L4G_Core/blob/master/src/scripts/scripts/zone/coilfang_resevoir/serpent_shrine/boss_lady_vashj.cpp#L189

Subsequent timer to 25-30sec from 120sec (spell has 20sec duration)

https://github.com/Looking4Group/L4G_Core/blob/master/src/scripts/scripts/zone/coilfang_resevoir/serpent_shrine/boss_lady_vashj.cpp#L534

and cutdown CC Players to 2 from 5

https://github.com/Looking4Group/L4G_Core/blob/master/src/scripts/scripts/zone/coilfang_resevoir/serpent_shrine/boss_lady_vashj.cpp#L240

and add CC immunity spellcomponents to 

SPELL_PERSUASION 38511 (Or fix SpellEffect #1)

https://github.com/Looking4Group/L4G_Core/blob/55e60444feb46fad253645b6c77da0f9f0f3ef5e/src/game/SpellAuras.cpp#L4795

**Comment by smkrus** [comment]
"9:11 first MC after ~ 15sec , 10:55 MC ends 11:01 New MC

11:44 end 11:48 new Question is Videospeed = x2?"
Don't understand this at all.
You're mixing PoVs??.... Look at just the rogue part
(rogue part is sped up 1.5x, Check 10 seconds root debuff=6 seconds of video)
MC lasts 20 seconds long ( can simply look at the rogue debuff when he gets MC)
MC #1 starts 10:03. MC #2 starts at 10:40 >> ~40 seconds * 1.5 = 60 seconds interval.

**Comment by AbaddonL4G** [comment]
Timer & player amount fixed.
Non CC missing.

**Comment by Pbeitelmann** [comment]
CC (almost) fixed; Immunity against disorient currently not working

**Comment by Netherfrost** [comment]
We just had a main tank AND offtank getting mind control. Prolly because she swapped target in 1 sec for when she multishotted & rooting.

**Comment by honeyhoney** [comment]
To clarify on @Netherfrost's comment; the player at the top of the threat list is ignored when calculating which 2 random raid members to mind control. As a result it is possible to have both the main and off-tank mind controlled if there is a dps/healer that has exceeded the main tanks threat (and this isn't always immediately obvious as you can have higher threat that the main tank, but still remain below pull aggro).

Interesting "feature" if you ask me :)

**Comment by smkrus** [comment]
It's fun being buffed and all but the timer should be ~60 seconds not 25-35.

**Comment by Selphius** [comment]
@smkrus 
Then it wouldn't be proper pre-nerf vashj which was on retail at 2.0, simple as that. So big no-no to any change in this regard. I would understand if the boss was unkillable like kael at 2.0, but not vashj...

**Comment by smkrus** [comment]
Don't understand your reply. World first kill shows 60 seconds timer not 25-35 as it is right now on Hellfire.

**Comment by Netherfrost** [comment]
The threat issue isn't tied to vashj in particular.. This is a general threat issues which happens every time there's a script which says something like, ignore -threat at top player, not the actual aggro target..  Don't know coding but I'm sure you know what I mean. I have trouble grasping why you would ignore a bug intentionally, naturally assuming it is one (?), tune the encounter all you want, but I doubt bugs should be considered a part of tuning. The late patch argument doesn't really work for TBC, rogues in 2.4.3 are the weakest considering ie removal of bugged WF, removal of 125% hemo, removal of weapon skills etc. I don't know where you got those 30% from :) 

Tldr,  tune it to be 500% more difficult if you want, but bugs shouldn't be part of it.

**Comment by smkrus** [comment]
I also hate the argument of "2.4.3 talents yadayda"
Just looked at all TBC patchnotes for my class (warlock) and having a hard time finding buffs...

**Comment by Selphius** [comment]
I really don't know where do you see 60s timer on MC... when i watch at this video what I see is: 10:40 -> paladin get's MC (MC lasts 15-20s) in this case it wears off at 10;56 AND NEXT MC starts at 11:00. Meaning 4-5s AFTER MC ended on paladin which started at 10;40, so where the hell is this another 55s without MC? Maybe I really am blind...

**Comment by AnonXS** [comment]
https://github.com/Looking4Group/L4G_Core/pull/2872/commits/1842659dca09ff6b07867fbec4c9709f9ea3a0c1
